src/config.rs

```diff
@@ -476,6 +476,8 @@ impl HtmlConfig {
 pub struct Playpen {
     /// Should playpen snippets be editable? Default: `false`.
     pub editable: bool,
+    /// display the copy button
```

Comment:
Uppercase the first letter might be better and more consistent across comments. Also, we love to document  default values especially when it is not the default of that type.
```suggestion
    /// Display the copy button. Default: `true`.
```